### PR TITLE
CHANGELOG for 0.20.0

### PR DESCRIPTION
**Pull request by angelozerr**
CHANGELOG for 0.20.0

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Change Log
 
+## [0.20.0](https://github.com/eclipse/lemminx/milestone/32?closed=1) (March 29, 2022)
```
Comment (rgrunber) [inline CHANGELOG.md:3]: Milestone 33.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+
+### Enhancements
+
+ * Fold closing tags too. See [#1178](https://github.com/eclipse/lemminx/issues/1178).
```
Comment (rgrunber) [inline CHANGELOG.md:7]: Closing tags should be included in the code folding range.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+
+### Bug Fixes
+
+ * Completion for prefix of attribute name. See [#1133](https://github.com/eclipse/lemminx/issues/1133).
+
+### Other
+
+ * Tests: Assertion is added for additional text edits of a completion i…. See [#1186](https://github.com/eclipse/lemminx/pull/1186).
```
Comment (rgrunber) [inline CHANGELOG.md:15]: "i..." should be "item"

**Review by rgrunber — COMMENTED** [review]
- I think https://github.com/eclipse/lemminx/pull/1183 can be added also.

**Comment by angelozerr** [comment]
> I think https://github.com/eclipse/lemminx/pull/1183 can be added also.

added.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,22 @@
...
+
+### Bug Fixes
+
+ * NPE on xsd datatype autocompletion in binary mode. See [#1189](https://github.com/eclipse/lemminx/issues/1189).
+ * Completion for prefix of attribute name. See [#1133](https://github.com/eclipse/lemminx/issues/1133).
+
+### Other
+
+ * Tests: Assertion is added for additional text edits of a completion i…. See [#1186](https://github.com/eclipse/lemminx/pull/1186).
+ * Update GraalVM version used in binary verification builds. See [#1158](https://github.com/eclipse/lemminx/issues/1158).
```
Comment (fbricon) [inline CHANGELOG.md:18]: Update GraalVM version used in binary verification builds to x.y.z

File: CHANGELOG.md (hunk 1 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed.

File: CHANGELOG.md (hunk 3 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed.

File: CHANGELOG.md (hunk 4 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed.

File: CHANGELOG.md (hunk 2 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed.

**Comment by angelozerr** [comment]
Thanks @fbricon @rgrunber for your review.

**Pull request merged**